directory.json

```diff
@@ -984,6 +984,10 @@
       "title": "Set up traceparent query tag within your application",
       "path": "/docs/opentelemetry/traceparent"
     },
+    "pii-filtering": {
```

Comment:
PII filtering only matters if users are using Log Insights, I believe - in that context, I wonder if we can put this under "/docs/log-insights/setup" 🤔
Also in general, this page doesn't mention about Log Insights but it would be nice to mention somewhere to make it clear?

Reply:
Good point. I'll mention Log Insights and move the page there.

Reply:
Although on second thought... if we capture stuff from, e.g., pg_stat_activity, PII filtering could apply as well, right? I'm not sure that we should put this under Log Insights if it either currently does apply or potentially could apply to data we collect from other sources.

On a related note, though, maybe we should note what sources of data that we capture _may_ contain PII. We do say

>When a third-party tool like pganalyze processes your database
>statistics and logs, it may involve handling data that includes this PII.

but maybe that's too vague? How about I add

>Specifically, pganalyze collects data that may include PII from Postgres logs
>and pg_stat_activity.

Reply:
I'm late to the party, but I disagree with putting this under Log Insights - as you note, some of the settings don't have anything to do with logs, and we usually get this question in a broader context (i.e. which sensitive data is in pganalyze, and how it can be filtered).

Is there a reason we can't put this under the pganalyze collector section?

Reply:
I put it in its own section since it affects what shows up in the app, but on second thought, so do most of the collector settings :upside_down_face: . I have not moved it under Log Insights due to the above realization. But I think you're right: I'll move it to `collector/pii-filtering.mdx`.

Reply:
Under the collector works for me 👍 I think it'd be nice to put that into ToC rather than being an orphan page

Reply:
I think that was in regards to me taking another look re: wording/how we present things - I can do that tomorrow.

Reply:
Yeah, I still plan to do that, but those are just mechanical changes. In terms of review, I'm mostly concerned about the accuracy of the Impact section.